packaging/package.sh

```diff
@@ -3,11 +3,15 @@ set -euxo pipefail
 
 BASEDIR=$(dirname "$0")
 OUTPUT=$1
-GOARCH=$(go env GOARCH)
+GOARCH=amd64
```

Comment:
i kept the name as `GOARCH`. but if others think its better to change then i'll update to `MARCH` or just `ARCH`

Reply:
Updated! PTALA